.Rbuildignore

```diff
@@ -14,3 +14,4 @@
 ^CITATION\.cff$
 ^data-raw$
 ^tools$
+^vignettes/figure$
```

Comment:
I don't think that's quite right, we want these figures to be included. OTOH, the `.orig` file should be ignored.

Reply:
When `^vignettes/figure$ is removed from `.Rbuildignore` the R CMD check returns a note asking if the folder is left over from knitr.

Reply:
You can change the `fig.path` value in `knitr::opts_chunk$set()` to save figs directly in the `vignettes/` folder, without subfolders.

Reply:
When I set the `fig.path = "vignettes/figure/" it saves the figures in the correct place, but by specifying the path, the path in the vignette to render the images changes to "vignettes/figure/" which means they cannot locate files.